src/wurf/context_msg_resolver.py

```diff
@@ -26,9 +24,10 @@ def resolve(self):
 
         :return: The path as a string.
         """
+        
 
         self.ctx.start_msg('{} {}'.format(
-            self.method, self.dependency.name))
+            self.dependency.resolver_method, self.dependency.name))
```

Comment:
I would like to propose an alternative start_msg format, just to make things understandable for the user. The current format looks confusing if the resolver_method comes first.
If it is an **active** step where we actually resolve something:
`Resolve "waf-tools" (git semver/git checkout/user path)`
I think we can keep Load for the passive step, but I am not fully happy with Load (help) :
`Load "waf-tools"` or `Load (help) "waf-tools"`

Of course, this would require that we define a separate `resolver_action` (Resolve/Load) and a `resolver_method` which is only needed for an active step.